sdk/storage/azure-storage-blob/azure/storage/blob/_shared/policies.py

```diff
@@ -241,6 +243,30 @@ def on_response(self, request, response, **kwargs):
                 _LOGGER.debug("Failed to log response: %s", repr(err))
 
 
+class StorageUserAgentPolicy(SansIOHTTPPolicy):
+
+    _USERAGENT = "User-Agent"
+
+    def __init__(self, **kwargs):
+        self._application = kwargs.pop('user_agent', None)
+        self._user_agent = "azsdk-python-storage.blob/{} Python/{} ({})".format(
```

Comment:
azsdk-python-storage**-**blob

Reply:
(replace the dot between storage and blob with a dash. The example in the general guidelines shows what it would look like for .NET. Which uses dots in their package names)